### PR TITLE
DAOS-4451 md: temporary logging for container/create failures

### PR DESCRIPTION
**Pull request by kccain**
Running soak stress testing of RPMs created from the release/0.9 branch
revealed (in a 48 hour period) two instances of container create
failures. Error logging is kept to a minimum for long stress test
runs like this. With this change, temporarily add D_ERROR logging to
the container create code paths to capture some preliminary information
on the nature of the failures.

Signed-off-by: Kenneth Cain <kenneth.c.cain@intel.com>

### CODE REVIEW

File: src/container/srv_container.c [hunk 1]
```diff
@@ -549,16 +549,20 @@ cont_create(struct rdb_tx *tx, struct ds_pool_hdl *pool_hdl,
...
+	}
 
 	/* Check if a container with this UUID already exists. */
 	d_iov_set(&key, in->cci_op.ci_uuid, sizeof(uuid_t));
 	d_iov_set(&value, NULL /* buf */, 0 /* size */);
 	rc = rdb_tx_lookup(tx, &svc->cs_conts, &key, &value);
 	if (rc != -DER_NONEXIST) {
 		if (rc == 0)
-			D_DEBUG(DF_DSMS, DF_CONT": container already exists\n",
+			D_ERROR(DF_CONT": container already exists\n",
```
Comment (liw) [inline src/container/srv_container.c:565]: Did you intend to do the following instead?

    if (rc != -DER_NONEXIST) {
        if (rc == 0)
            D_DEBUG(...);
        else
            D_ERROR(...);
        D_GOTO(out, rc);
    }

**Comment by kccain** [comment]
The companion PR 2446 has landed on daos master branch.

Landing this PR today on release/0.9 branch is a nice-to-have, so that this weekend's soak testing may benefit from daos container create error logging, should any very intermittent failures occur as they have in prior test executions.

**Comment by jolivier23** [comment]
@kccain Now that 1.0 is released, should we abandon this?

**Comment by kccain** [comment]
@jolivier23 yes we can abandon this PR and branch since the logging has been landed to master. The DAOS-4451 ticket has been reopened and scheduled for resolution in DAOS-1.2 so any of that work would be based on master and (if applicable to whatever the actual problem is) this logging is available.

**Pull request closed without merging**